packages/appstore/src/components/onboarding-new/static-cfd-account-manager.tsx

```diff
@@ -64,8 +64,8 @@ const StaticCFDAccountManager = ({
     is_onboarding_animated,
     is_eu_user,
 }: TStaticCFDAccountManager) => {
-    const icon_size = isMobile() ? 48 : 48;
-    const icon_size_derivx = isMobile() ? 48 : 48;
+    const icon_size = 48;
```

Comment:
you can pass this number directly to the component, adding a variable is redundant

Reply:
I added a variable so that we do not have to change it statically just like my last commit.
I changed the value by just changing the value of the variable instead of changing it in 8 other places.